lib/query/builder.js

```diff
@@ -1008,36 +1008,45 @@ assign(Builder.prototype, {
     return this;
   },
 
-  // Remove everything from select clause
+  // Deprecated. Remove everything from select clause
```

Comment:
Can you please add non-deprecated alternative in comments to every deprecated method?

Reply:
@elhigu Do you think we should output deprecation warning here as well?

Reply:
I think those separate clear methods can be deprecated as long as list of valid string literals are listed in typings.